routes/download.js

```diff
@@ -0,0 +1,140 @@
+const { Readable } = require('stream');
+
+const express = require('express');
+const AWS = require('aws-sdk');
+const zlib = require('zlib');
+
+
+AWS.config.update({ region: 'us-east-1' });
+const s3 = new AWS.S3();
+
+const router = express.Router();
+require('dotenv').config();
+
+// handles when missing release_id
+router.get('/', (req, res) => {
+  return res.status(400).json({
+    message: 'missing a required field',
+  });
+});
+
+router.get('/:release_id', (req, res) => {
+  const { release_id } = req.params;
+  const params = {
+    Bucket: SNAPSHOT_BUCKET,
+    Prefix: release_id,
+  };
+
+  listObjectsAsync(params)
+    .then((body) => {
+      // handles if a release does not exist
+      if (body.Contents.length === 0) {
+        return res.status(404).json({
+          message: 'release not found',
+        });
+      }
+
+      return getObjectsAsync(params, body.Contents);
+    })
+    .then((obj) => {
+      // set response headers
+      res.attachment(`${release_id}.gz`);
```

Comment:
Think this needs to be updated to `.json.gz`. When uncompressed, it will make a file without an extension.

Reply:
@dankolbman That's correct. I will change `${release_id}.gz` to `${release_id}.json.gz`

Reply:
@dankolbman Updated it as kicking in downloading RE_00000001.json.gz in a well-formatted JSON.